src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1061,6 +1067,18 @@ IS 'directly sourced from moj_transcription_r';
 COMMENT ON COLUMN transcription_comment.transcription_object_id
 IS 'internal Documentum id from moj_transcription_s acting as foreign key';
 
+CREATE TABLE transcription_status

```

Comment:
Are we missing the standing data inserts here?

Reply:
I was expecting to see the INSERT statements for the 5 `transcription_status` we want?

Reply:
Judith has added to standing data script